.circleci/config.yml

```diff
@@ -59,6 +60,29 @@ jobs:
           name: DevOps checkout
           command: |
             git clone $DEVOPS_REPO $DEVOPS_WORKDIR
+      - run:
+          name: Start dev server
+          command: |
+            cp $DEVOPS_WORKDIR/environments/cypress .env
```

Comment:
We might need `$DEVOPS_WORKDIR/environments/local` here instead. The cypress configuration has not been updated for a while

Reply:
Good call on using something different. I just browsed the `local` config, and I think it might provide too many "real" connections for a testing environment. I'll open a PR over there to create a test env.